metrics/multilabelmap.go

```diff
@@ -13,6 +13,13 @@ import (
 	"sync"
 )
 
+// metricType is a Prometheus metric type. At the moment we only support
+// counter and gauge.
+type metricType string
+
+const Counter metricType = "counter"
+const Gauge metricType = "gauge"
```

Comment:
exported things with unexported types is kinda weird

Reply:
I've exported the type as well.